packages/explat-client/src/internal/experiment-assignments.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Internal dependencies
+ */
+import { ExperimentAssignment } from '../types';
```

Comment:
If you're importing just a type you can use `import type` to help speed up type erasure.